packages/strapi-utils/lib/buildQuery.js

```diff
@@ -105,12 +100,10 @@ const normalizeFieldName = ({ model, field }) => {
  * @param {Object} options.filters - The filters for the query (start, sort, limit, and where clauses)
  * @param {Object} options.rest - In case the database layer requires any other params pass them
  */
-const buildQuery = ({ model, filters = {}, ...rest }) => {
+const buildQuery = ({ model, filters = {}, search = {}, ...rest }) => {
```

Comment:
I don't think you should change this function. are you don't use the search params at all. it wil be passed with the `...rest`

Reply:
You can move the search = {} directly into the queries =)